src/Makemodule.am

```diff
@@ -135,14 +136,14 @@ TESTS = src/czmq_selftest
 
 # Run the selftest binary under valgrind to check for memory leaks
 memcheck: src/czmq_selftest
-	$(LIBTOOL) --mode=execute valgrind --tool=memcheck \
+	libtool --mode=execute valgrind --tool=memcheck \
 		--leak-check=full --show-reachable=yes --error-exitcode=1 \
 		--suppressions=$(srcdir)/src/.valgrind.supp \
 		$(srcdir)/src/czmq_selftest
 
 # Run the selftest binary under gdb for debugging
 debug: src/czmq_selftest
-	$(LIBTOOL) --mode=execute gdb -q \
+	libtool --mode=execute gdb -q \
```

Comment:
Not sure why this change was made - maybe by accident?  I recall that on some systems, using `$(LIBTOOL)` instead of `libtool` was important, perhaps when there is more than one libtool, or the libtool to use was in a non-standard path.

I'd suggest this be reverted unless there is a good reason.